docs/md_merge.py

```diff
@@ -0,0 +1,357 @@
+import argparse
+import os
+import re
+import shutil
+from argparse import BooleanOptionalAction
```

Comment:
I guess that won't work for Python 3.8. We should at least document that we need Python 3.9, as Focal Fossa has still nearly 3 years support...

**Edit**: I should have read the PR description first... Nevertheless, we should document that anyway.

Reply:
I added some more docstrings to the script and included a note on python 3.9.